convert2rhel/breadcrumbs.py

```diff
@@ -67,23 +67,25 @@ def __init__(self):
 
     def collect_early_data(self):
         """Set data which is accessible before the conversion"""
-        self._set_pkg_object()
-        self._set_executed()
-        self._set_nevra()
-        self._set_signature()
-        self._set_source_os()
-        self._set_started()
+        if not "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
```

Comment:
This is not needed, as it will stop the generation of `/etc/migration-results` file as well.